WeatherApp/DAL/Models/OpenWeather/Wind.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
+
+namespace DAL.Models.OpenWeather
+{
+    public class Wind
+    {
+        [JsonPropertyName("speed")]
```

Comment:
Json serialization\deserialization is case insensitive. I may skip attribute if difference is just a case

Reply:
JsonPropertyName attributes have been removed.